docs/content/docs/build/containers.md

```diff
@@ -3,28 +3,153 @@ title: Containers
+
 ### Deleting a container
 
-## Lifecycle
+Deleting a container is a service-specific feature, so you should refer to the documentation associated with the specific Fluid service you are using. See [Available Fluid Services]({{< relref "service-options.md" >}}) for more information about Fluid service options.
+
+## Container lifecycle and manipulation
+
+### connected/disconnected
+
+The container exposes the connected state of the client and emits connected and disconnected events to notify the caller if the underlying connection is disrupted. Fluid will by default attempt to reconnect in case of lost/intermittent connectivity.
+
+```typescript
+const connected = container.connected();
+
+container.on("disconnected", () => {
+    // handle disconnected
+    // prevent the user from editing to avoid data loss
+});
+
+container.on("connected", () => {
+    // handle connected
+    // enable editing if disabled
+});
+```
+
+### dispose
 
-### connected/disconnected/disposed
+The container object may be disposed to remove any server connections and clean up registered events. Once a container is disposed, you must call `getContainer` again if you want to re-load it.
+
+```typescript
+container.dispose();
+
+const disposed = container.disposed();
+
+container.on("disposed", () => {
+    // handle event cleanup to prevent memory leaks
+});
+```
+
+### initialObjects
+
+`initialObjects` are shared objects that you define in a container's schema and exist for the lifetime of the container. These shared objects are exposed via the `initialObjects` property on the container.
+
+```typescript
+const schema = {
+    name: "example-container",
+    initialObjects: {
+        layout: SharedMap,
+        text: SharedString
+    },
+}
+
+// ...
+
+const layout = container.initialObjects.layout;
+const text = container.initialObjects.text;
+```
+
+For more information about `initialObjects` see [Data modeling](data-modeling.md).
+
+### create
+
+The container also exposes a create function that allows dynamic creation of shared objects. This enables containers to create Fluid objects dynamically at runtime.
+
+For more information about dynamic object creation see [Data modeling](data-modeling.md).
 
 ## Patterns for managing container lifecycle
 
-### Create should be separate from load
+### Create/load separation
+
+When creating and loading a container, it can be tempting to have a consistent code path for both creation and loading. However, in Fluid it is generally better to separate the two flows. This provides a cleaner separation of responsibilities within the code itself. Also, in typical use-cases, a user will create a new container through some UI action that results in a redirect to another page whose sole responsibility is to load a container. All subsequent users will load the container by navigating directly to that page.
```

Comment:
```suggestion
When creating and loading a container, it can be tempting to have a consistent code path for both creation and loading. However, it is generally recommended to separate the two flows. This provides a cleaner separation of responsibilities within the code itself. Also, in typical use-cases, a user will create a new container through some UI action that results in a redirect to another page whose sole responsibility is to load a container. All subsequent users will load the container by navigating directly to that page.
```